demo/api/views/cars.py

```diff
@@ -20,15 +20,15 @@ def get(request: Request, version: int, **kwargs) -> Response:
         return Response(cars, 200)
 
     @staticmethod
-    def put(request: Request, version: int, vehicle_type: str) -> Response:
```

Comment:
this apiview is badly designed (i know the demo project comes from the drf-yasg repo) and it failed schema validation because, well, this is a list endpoint that never uses the path parameter.